From d5f5b15f5d10b4d6430a05be88f751ff6b33e5a8 Mon Sep 17 00:00:00 2001
From: Zelan Zou <zelan.zou@nxp.com>
Date: Fri, 10 Oct 2025 17:40:03 +0800
Subject: [PATCH] mxc_uart_xmit_test.c: Fix termio.h replaced by termios.h

| mxc_uart_test/mxc_uart_xmit_test.c:21:10: fatal error: termio.h: No such file or directory
|    21 | #include <termio.h>
|       |          ^~~~~~~~~~
Upstream-Status: Inappropriate [Pending]

Signed-off-by: Zelan Zou <zelan.zou@nxp.com>
---
 test/mxc_uart_test/mxc_uart_xmit_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_uart_test/mxc_uart_xmit_test.c b/test/mxc_uart_test/mxc_uart_xmit_test.c
index 7077718..6b5a400 100644
--- a/test/mxc_uart_test/mxc_uart_xmit_test.c
+++ b/test/mxc_uart_test/mxc_uart_xmit_test.c
@@ -18,7 +18,7 @@
  */
 
 #include <stdarg.h>
-#include <termio.h>
+#include <termios.h>
 #include <sys/timeb.h>
 #include <pthread.h>
 #include <stdio.h>
-- 
2.34.1

